### PR TITLE
Array Rolation in python

### PR DESCRIPTION
**Pull request by soumikbaithalu**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Let's take an example that given the array [1,2,3,4,5,6,7] so I have to rotate the left side at particular side 2 to 7 so the output is [3,4,5,6,7,1,2].

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

This is an issue related to #4098

### CODE REVIEW

File: Python/other/Program_to_rotation_array.py
Comment (siddharth25pandey) [inline Python/other/Program_to_rotation_array.py]: Write a good naming convention

File: Python/other/Program_to_rotation_array.py
Comment (siddharth25pandey) [inline Python/other/Program_to_rotation_array.py]: Display a message in print form as what are you taking input from the user for better UI

File: Python/other/Program_to_rotation_array.py
Comment (siddharth25pandey) [inline Python/other/Program_to_rotation_array.py]: Display a message in print form as what are you showing in output

Comment (siddharth25pandey) [inline Python/other/Program_to_rotation_array.py]: At the end write Time & Space Complexity of the given Algorithm
And also write Sample Input and output in commented form

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@soumik2012 Do the suggested changes

File: Python/other/Program_to_rotation_array.py
Comment (soumikbaithalu) [reply Python/other/Program_to_rotation_array.py]: Please check my code c269bdb

File: Python/other/Program_to_rotation_array.py
Comment (soumikbaithalu) [reply Python/other/Program_to_rotation_array.py]: Please check my code c269bdb

Comment (soumikbaithalu) [reply Python/other/Program_to_rotation_array.py]: As you requested I changes made please check c269bdb @siddharth25pandey

**Review by siddharth25pandey — APPROVED** [review]
@soumik2012 LGTM 💯 Able to merge, after resolving the merge conflicts error

**Comment by siddharth25pandey** [comment]
@soumik2012 Resolve the merge conflict error

**Comment by soumikbaithalu** [comment]
@siddharth25pandey sir  merge conflict error is resolve can merge it

**Review by siddharth25pandey — APPROVED** [review]
@soumik2012 LGTM 💯

**Comment by siddharth25pandey** [comment]
@all-contributors please add @soumik2012 for his code

**Pull request merged**
